src/test/java/pages/RegistrationPage.java

```diff
@@ -0,0 +1,100 @@
+package pages;
+
+import elements.Button;
+import elements.Checkbox;
+import elements.Input;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import util.WaitUtils;
+
+public class RegistrationPage extends BasePage {
+
+    @FindBy(xpath = "//input[@type='email']")
+    protected Input loginInputField;
+
+    @FindBy(xpath = "//div[6]/descendant::input")
+    protected Input passwordInputField;
+
+    @FindBy(xpath = "//div[@autocomplete='repeatPassword']/descendant::input")
+    protected Input repeatPasswordInputField;
+
+    @FindBy(xpath = "//span[@class='i-checkbox__faux']")
+    protected Checkbox privacyPolicyCheckbox;
+
+    @FindBy(xpath = "//button[@type='submit']")
+    protected Button registrationButton;
+
+    @FindBy(css = ".auth-container .auth-form__title ")
+    protected Button emailConfirmationRequest;
+
+    @Override
+    public RegistrationPage openPage() {
+        driver.navigate().to(props.getKeyProperty("regURl"));
+        logger.debug("Navigation to the url...");
+        return this;
+    }
+
+    @Override
+    public RegistrationPage isPageOpened() {
+        try {
+            WaitUtils.waitForVisibility(loginInputField);
+        } catch (TimeoutException e) {
+            logger.debug("Element isn't found...");
```

Comment:
what element? 
please see the comment above and check all the similar cases

Reply:
fixed